.travis.yml

```diff
@@ -0,0 +1,9 @@
+
+language: go
+go: 
+ - tip
+
+script:
+  - gofmt -d -s $(find . -name '*.go')
+  - test -z "$(gofmt -d -s $(find . -name '*.go'))"
```

Comment:
The first run is to show the developer what the formatting problem is, if any.
The second run is to allow picking up on if there was any output and fail the build accordlingly, because `go fmt` always seems to exit with a 0.